lib/fluent/plugin/kafka_plugin_util.rb

```diff
@@ -42,6 +46,13 @@ def read_ssl_file(path)
         end
       end
 
+      END_CERTIFICATE = "\n-----END CERTIFICATE-----\n"
+
+      def read_ssl_ca_certs
+        return nil if @ssl_ca_cert.nil?
+        @ssl_ca_cert.reject { |p| invalid_path?(p) }.flat_map { |c| File.read(c).split(END_CERTIFICATE).map { |c| c += END_CERTIFICATE} }
+      end
+
```

Comment:
It may be cosmetic feedback, but it is readable not to use oneliner aggressively.

```
valid_paths = @ssl_ca_cert.reject { |path| invalid_path?(path) }
valid_paths.flat_map do |path|
  File.read(path).split(END_CERTIFICATE).map { |c| c += END_CERTIFICATE}
end
```

Reply:
Corrected. Thanks!